erpnext/selling/doctype/customer/customer.js

```diff
@@ -116,14 +116,23 @@ frappe.ui.form.on("Customer", {
 			frappe.contacts.render_address_and_contact(frm);
 
 			// custom buttons
-			frm.add_custom_button(__('Accounting Ledger'), function() {
-				frappe.set_route('query-report', 'General Ledger',
-					{party_type:'Customer', party:frm.doc.name});
-			});
+			// frm.add_custom_button(__('Accounting Ledger'), function() {
+			// 	frappe.set_route('query-report', 'General Ledger',
+			// 		{party_type:'Customer', party:frm.doc.name});
+			// });
+
+			// frm.add_custom_button(__('Accounts Receivable11'), function() {
+			// 	frappe.set_route('query-report', 'Accounts Receivable11', {customer:frm.doc.name});
+			// });
```

Comment:
```suggestion
```

Commenting out code is never required. Feel free to delete stuff that's not required.